Cloud/Docker/build.sh

```diff
@@ -36,11 +36,10 @@ do
         jerryscript)
             BUILD_JERRYSCRIPT=true
             ;;
-        all)
+        major)
             BUILD_JSC=true
             BUILD_SPIDERMONKEY=true
             BUILD_V8=true
-            BUILD_DUKTAPE=true
             ;;
         *)
             echo "Usage: $0 [fuzzilli|jsc|spidermonkey|v8|duktape|jerryscript|all]"
```

Comment:
Nit: please s/all/major/ here :)

Reply:
I changed it in the script, and in the Readme!